api/controllers/v0.3/endpoints.js

```diff
@@ -397,15 +393,10 @@ async function createPatron(req, res) {
   // etc., or if the username is unavailable, render that error.
   if (errors && Object.keys(errors).length !== 0) {
     response = {
-      ...modelResponse.errorResponseData(
-        collectErrorResponseData(
-          400,
-          "invalid-request",
-          "There was an error with the request.",
-          "",
-          errors
-        ) // eslint-disable-line comma-dangle
+      ...collectErrorResponseData(
+        new InvalidRequest("There was an error with the request.")
```

Comment:
There are still some places where the error is not coming from a controller function but rather directly here. Eventually I want to update these.